### DISCUSSION TITLE
Create a discord server

### DISCUSSION THREAD

**Discussion by OhadBaehr** [kuma-ui/kuma-ui#111] [Ideas]
I think a discord server would be extremely beneficial to accelerate this project's progress, just found out about it today, I think this project is super important for the ecosystem and I'm looking forward to helping out.

**Comment by poteboy** [comment]
Thank you so much for your kind words, it means a lot to me! I agree, a Discord server would be very helpful for this project. I'll set one up soon, so please hang tight!

**Reply by poteboy** [reply]
FYI  @harshsinghatz

**Comment by poteboy** [comment]
We've created a Discord server for Kuma. Please check this out and come join us 🐻‍❄️  https://discord.gg/QrsQ4EPp7G
